src/pages/seed-exchange/index.jsx

```diff
@@ -1,9 +1,47 @@
 import Head from "next/head";
-import React from "react";
+import React, { useState } from "react";
 import Layout from "../../organisms/layout";
 import Typography from "@mui/material/Typography";
+import axios from "axios";
+import { Input } from "@mui/material";
+import PhotoCamera from "@mui/icons-material/PhotoCamera";
+import Box from "@mui/material/Box";
+import IconButton from "@mui/material/IconButton";
+import styled from "@emotion/styled";
+
+const CLOUDNAME = process.env.NEXT_PUBLIC_CLOUDNAME;
+const PRESET = process.env.NEXT_PUBLIC_PRESET;
 
 const Page = () => {
+	const [image, setImage] = useState(null);
+
+	const Image = styled.img`
+		max-width: 40%;
+		max-height: 40%;
+		border: 4px solid #e4eaeb;
+		border-radius: 50%;
+	`;
```

Comment:
same here, if you dont put it outside it will be rendered a lot.. during every render-cycle